specification/cognitiveservices/data-plane/QnAMaker/readme.md

```diff
@@ -0,0 +1,41 @@
+# Cognitive Services QnAMaker Api SDKs
+
+> see https://aka.ms/autorest
+
+Configuration for generating QnAMaker Api SDK.
+
+``` yaml
+tag: QnAMaker
+add-credentials: true
+openapi-type: data-plane
+```
+
+The current release for the QnAMaker Api is `4.0`.
+
+## Swagger to SDK
+
+This section describes what SDK should be generated by the automatic system.
+This is not used by Autorest itself.
+
+### QnAMaker Api- CSharp Settings
+These settings apply only when `--csharp` is specified on the command line.
+
+``` yaml $(csharp)
+csharp:
```

Comment:
should indicate the location of swagger file. otherwise the autorest won't work

Reply:
used face readme, let me know if its fine now